plaso/output/shared_elastic.py

```diff
@@ -323,6 +324,9 @@ def _InsertEvent(self, event, event_data, event_data_stream, event_tag):
     event_values = self._GetSanitizedEventValues(
         event, event_data, event_data_stream, event_tag)
 
+    if self._additional_fields:
+      event_values.update(self._additional_fields)
```

Comment:
I think we have to make sure there are no duplicate field names here

Reply:
again, this is a draft PR, and I'm just testing this, this is not the final version